delivery/kustomize/base/deployment-hat.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     app.kubernetes.io/name: podtato-head
     app.kubernetes.io/component: hat
-    app.kubernetes.io/version: '0.2.1'
+    app.kubernetes.io/version: '0.2.3'
     app.kubernetes.io/managed-by: kustomize
 spec:
   replicas: 1
```

Comment:
This shows that you did not use operator-sdk to built the operator. Am I right?
WDYT about use the tool?

Reply:
Hi @camilamacedo86, thanks for taking a look. There are a few different tools illustrated in this repo in the `./delivery` dir, all delivering the same system from `podtato-head-microservices`. In this PR I used operator-sdk in the test scripts and README to pick up the Helm chart from the example Helm scenario in `./delivery/chart` and generate and deploy a Helm operator based on it.

I didn't actually check anything in that was generated by the SDK, it's used statelessly you might say. What do you think of that?